scripts/procedures/properties.ts

```diff
@@ -350,35 +356,46 @@ export const properties: T.ExpectedExports.properties = async (
   const result: T.Properties = {
     version: 2,
     data: {
-      "Node Uri": {
+      "Node ID": {
```

Comment:
Lines 359 - 368, 371,372 were the only ones really changed.

Everything else is just prettier acting like it was doing something.